mu-trees/addon/ember-config.js

```diff
@@ -18,12 +18,14 @@ export default function generateConfig(name) {
       controller: { definitiveCollection: 'routes' },
       component: { definitiveCollection: 'components' },
       'component-lookup': { definitiveCollection: 'main' },
+      'component-manager': { definitiveCollection: 'main' },
```

Comment:
Hmm, I don't think `main` is right? Shouldn't it be `component-managers` (assuming `src/component-managers/*` is where they go)?

Reply:
That makes sense yeah. I haven't tested that myself yet though. In https://emberclear.io I've been using main with sparkles.
I can do some testing tomorrow

Reply:
so if I _just_ add
```
  'component-manager': { definitiveCollection: 'component-managers' },
```

I get this error with sparkles:
```
vendor.js:96748 Uncaught TypeError: Cannot read property 'defaultType' of undefined
    at RequireJSRegistry._checkDefaultType (vendor.js:96748)
    at RequireJSRegistry._detectModule (vendor.js:96737)
    at RequireJSRegistry.has (vendor.js:96757)
    at Resolver._serializeAndVerify (vendor.js:66164)
    at Resolver.identify (vendor.js:66132)
    at Resolver.resolve (vendor.js:66152)
    at Class.resolve (vendor.js:97505)
```

the resolver still doesn't 
find the component-manager, so... I needed to add to collections.

Reply:
You'd also need to update the defaultType